contracts/interfaces/WOOFiDex/INonceCounter.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.14;
+
+interface INonceCounter {
+    /* ----- Events ----- */
+
+    event CrossChainRouterUpdated(address crossChainRouter, bool flag);
+
+    /* ----- Functions ----- */
+
```

Comment:
As an interface, for the API's completeness, maybe add 
`inboundNonce` function ?